controllers/activemqartemis_controller.go

```diff
@@ -51,22 +49,8 @@ func GetBrokerConfigHandler(brokerNamespacedName types.NamespacedName) (handler
 }
 
 func UpdatePodForSecurity(securityHandlerNamespacedName types.NamespacedName, handler common.ActiveMQArtemisConfigHandler) error {
-	success := true
-	for nsn, fsm := range namespacedNameToFSM {
-		if handler.IsApplicableFor(nsn) {
-			fsm.SetPodInvalid(true)
-			glog.Info("Need update fsm for security", "fsm", nsn)
-			if err, _ := fsm.Update(); err != nil {
```

Comment:
This may need more work, and tests. The Update is a reconcile, but it may be what is necessary is a pod restart.

Reply:
sorted this with events chan source to force reconcile of the matching cr's